test/integration/fixtures/blocks/core__media-text__image-fill-no-focal-point-selected.html

```diff
@@ -1,9 +1,8 @@
-<!-- wp:media-text {"mediaType":"image","imageFill":true} -->
-<div class="wp-block-media-text alignwide is-image-fill is-stacked-on-mobile">
-	<figure class="wp-block-media-text__media"
-		style="background-image:url(data:image/jpeg;base64,/9j/2wBDAAMCAgICAgMCAgIDAwMDBAYEBAQEBAgGBgUGCQgKCgkICQkKDA8MCgsOCwkJDRENDg8QEBEQCgwSExIQEw8QEBD/yQALCAABAAEBAREA/8wABgAQEAX/2gAIAQEAAD8A0s8g/9k=);background-position:50% 50%">
+<!-- wp:media-text {"align":"wide","mediaType":"image","imageFill":true} -->
```

Comment:
These are manual changes from you, right?
Fixtures for deprecations consist of 4 files and we write the 'main' html one and the rest are created from the update fixtures script.

When we're updating a block and need a deprecation, this main fixture shouldn't change at all, with the sole exception and if needed the main basic fixture for the block. I'm saying main, because we have fixtures:
1. Current block markup
2. All deprecation that exist


So, if you needed manual updates, it seems the migration isn't correct.

Reply:
Ah, I see, thank you. My goal was to keep the non-migration main fixtures up-to-date with the current output, but you're right that we should probably keep them as-is for additional validation.

I do think that some of them were failing and that's why I decided to update them in the first place, so I'll investigate 👍

Reply:
So it turns out I *do* need to update the base fixtures, since only "deprecated" features are allowed to result in updated blocks. If a fixture without "__deprecated" in the file name results in an updated block, the tests fail because of a `console.info()` message being emitted.

So with ac0ea5d35e0f659d54122d0b7fac1e3c8f4bf480 I'm doing both; keeping the updated code in the base fixtures to ensure that we test the new markup, but also adding back the old versions as new `__deprecated` fixtures, to ensure we test those upgrades.

Does that work for you, @ntsekouras?

Reply:
Oh, yes. I forgot about that detail. Sorry for that! Maybe not all the new deprecations are really needed, but they definitely not hurt. Thank you!